lib/src/cli/atlas_apps/baas_client.dart

```diff
@@ -353,6 +355,16 @@ class BaasClient {
     }
   }
 
+  Future<String> createApiKey(BaasApp app, String name, bool enabled) async {
+    final dynamic result = await _post('groups/$_groupId/apps/${app.appId}/api_keys', '{ "key":"", "name":"$name", "disabled": false }');
+    if (!enabled) {
+      // We need to disable the key with another API call - the argument in the create call is ignored.
```

Comment:
Is it a bug in Atlas API?

Reply:
Maybe it's intentional - I simplified the original call though and removed that comment.